Foundation/NSPathUtilities.swift

```diff
@@ -30,7 +30,11 @@ public func NSTemporaryDirectory() -> String {
             return tmpdir
         }
     }
+    #if os(Android)
```

Comment:
/data/local/tmp isn’t writable by an app anyway so I’ve reverted this change. You have to set TMPDIR env var to the Content.getCacheDir() for various things to work.